pkg/proxy/iptables/proxier.go

```diff
@@ -289,18 +289,23 @@ func NewProxier(ipt utiliptables.Interface,
 
 	serviceHealthServer := healthcheck.NewServiceHealthServer(hostname, recorder)
 
-	isIPv6 := ipt.IsIPv6()
+	ipFamily := v1.IPv4Protocol
```

Comment:
In general, I find this pattern of having an `ipFamily` variable easier to read then a `isIPv6` bool var. Would be great if we can use this consistently in the code base (in future PRs). Maybe it's just me though.

Reply:
it is. I have also modified the way we test for IPv6 `(ipFamily == v1.IPv6Protocol) == utils.IsIPv6String(<IP>)`

Reply:
We can do some followup to be more consistent.  Agree with the point, but every time this PR changes my hair goes greyer.